### PR TITLE
Create PlayRoyal.com.fc

### PR DESCRIPTION
**Pull request by aviramdayan**
Create

### CODE REVIEW

File: PlayRoyal.com/PlayRoyal.com.fc [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+
+      //=============//
+      //FaucetClaim
+      //=============//
+      if (GetBoolSetting("FaucetClaim"))
+      {
+
+        //go to CancelSecurity FaucetClaim page
+		GoToUrl("https://playroyal.com/");
+        Log("wait 30 sec - to get daily token ");
```
Comment (faucetcollector) [inline PlayRoyal.com/PlayRoyal.com.fc]: It just have to wait to get the claim?

File: PlayRoyal.com/PlayRoyal.com.fc [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+//=============//
+//DoSolveFaucet
+//=============//
+  public override int DoSolveFaucet()
+  {
+
+      //=============//
+      //FaucetClaim
+      //=============//
+      if (GetBoolSetting("FaucetClaim"))
```
Comment (faucetcollector) [inline PlayRoyal.com/PlayRoyal.com.fc]: I don't think anyone wants this option unchecked, you can remove it.

File: PlayRoyal.com/PlayRoyal.com.fc (hunk 1 above)
Comment (aviramdayan) [reply PlayRoyal.com/PlayRoyal.com.fc]: yeah

File: PlayRoyal.com/PlayRoyal.com.fc (hunk 2 above)
Comment (aviramdayan) [reply PlayRoyal.com/PlayRoyal.com.fc]: i keep

**Pull request closed without merging**